.github/workflows/node.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 12, 14]
+        node: [14, 16, 18]
```

Comment:
I had an issue installing the dependencies since 10 and 12 reached their EOL if I am not mistaken https://nodejs.org/en/about/releases/